DuckDuckGo/HomePageConfiguration.swift

```diff
@@ -24,16 +24,27 @@ class HomePageConfiguration {
     
     enum ConfigName: Int {
 
-        var components: [Component] {
+        func components(withVariantManger variantManger: VariantManager = DefaultVariantManager()) -> [Component] {
```

Comment:
I'm somewhat confused by this section. Are the cases meant to represent independent home page configurations that a user can have? Looking at `centerSearchAndFavorites` it looks that way. Should we add the privacy section as a new case?

Reply:
We discussed this over zoom and this now makes sense as they are not discrete options.